src/makeCalculator.js

```diff
@@ -4,7 +4,51 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (x) {
```

Comment:
Fix it everywhere 
<img width="1137" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/64750958/5100064d-61f9-4a60-81e9-83f3f1245393">